spec/available-package-view-spec.coffee

```diff
@@ -42,7 +52,7 @@ describe "PackageCard", ->
     view.installButton.click()
     expect(@packageManager.install).toHaveBeenCalled()
 
-  it "hides the settings button if a package has no settings", ->
+  it "removes the settings button if a package has no settings", ->
     setPackageStatusSpies {installed: true, disabled: false, hasSettings: false}
     view = new PackageCard {name: 'test-package'}, @packageManager
     expect(view.find('.btn.settings')).not.toExist()
```

Comment:
Whoops, I didn't realize you could do `view.find.settingsButton` so I did it the hard way here – we should probably change this.